sycl/test-e2e/CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ find_package(Python3 REQUIRED COMPONENTS Interpreter)
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/lit.site.cfg.py.in"
                "${CMAKE_CURRENT_BINARY_DIR}/lit.site.cfg")
 
+list(APPEND SYCL_E2E_TEST_DEPS
+  sycl-toolchain
+  )
+
```

Comment:
I think we to guard that to skip it in standalone mode (see line 7 for its definition).

Reply:
Good point. I added the guard.